packages/ui/src/helpers/authenticator/facade.ts

```diff
@@ -171,6 +169,8 @@ export const getServiceContextFacade = (
         return 'verifyUser';
       case actorState?.matches('confirmVerifyUser'):
         return 'confirmVerifyUser';
+      case state.matches('signIn.runActor'):
+        return 'transition';
```

Comment:
It's odd to me that the rest of these `case` statements are matching against route values and this one does not/ What do we need to align with the existing statements?

Reply:
The `signIn.runActor` is a catch all state for the `signIn` actor when it's running. The `transition` route is returned only if all other states, and actor states in the switch statement return false.

For this change we care about two different actor states when the `signIn.runActor` is running. The first state is the `autoSignIn.pending` and the `resolved` state. The problem is covering both states, adding in `autoSignIn.pending` can be done like this:

```
  case actorState?.matches('autoSignIn'):
        return 'transition';
```

What's more tricky is covering the `resolved` actor state, since the string `resolved` is used in all our other actor states, not just `signIn`. If we add in a switch statement for the actor state for resolved, it would catch when any other actor state was in the `resolved` state (like signUp).

This was the best solution I came up with for now, without refactoring the xState machine more, or renaming more states.

Reply:
Thanks for explaining. Can we please add a comment here summarizing what you have written above?

Reply:
OK, I'll leave a comment about why this route is in there.